apps/demos/Demos/DataGrid/ToolbarCustomization/ReactJs/App.js

```diff
@@ -80,7 +80,7 @@ const App = () => {
             <span>Total Count</span>
           </div>
         </Item>
-        <Item location="before">
+        <Item location="before" locateInMenu="auto">
```

Comment:
Task "Demos Frameworks / Check generated demos" is failed, it checks that ReactJS demo is up-to-date with React demo
Let's see what's wrong in personal